backend/kesaseteli/applications/jobs/daily/clean_old_employer_applications.py

```diff
@@ -1,13 +1,21 @@
+import logging
+
 from dateutil.relativedelta import relativedelta
 from django.utils import timezone
 from django_extensions.management.jobs import DailyJob
 
 from applications.models import EmployerApplication
 
+LOGGER = logging.getLogger(__name__)
+
 
 class Job(DailyJob):
-    help = "Clean applications older than five years."
+    help = "Clean employer applications older than five years."
 
     def execute(self):
+        LOGGER.info("Clean employer applications older than five years...")
         five_years_from_now = timezone.now() - relativedelta(years=5)
-        EmployerApplication.objects.filter(created_at__lte=five_years_from_now).delete()
+        result = EmployerApplication.objects.filter(
+            created_at__lte=five_years_from_now
+        ).delete()
+        LOGGER.info(f"Cleaned {result[0]} employer applications.")
```

Comment:
As the delete function "returns the number of objects deleted and a dictionary with the number of deletions per object type" (See https://docs.djangoproject.com/en/3.2/topics/db/queries/#deleting-objects) the deleted count could be also unpacked using tuple unpacking and the throw-away variable `_` (This is a language level supported variable unlike in Javascript where it is simply convention but not language support AFAIK):
```
deleted_count, _ = EmployerApplication.objects.filter(
...
```

Reply:
Nice! Fixed now too.